pkg/server/skydb/pq/schema_test.go

```diff
@@ -28,7 +28,7 @@ func TestExtend(t *testing.T) {
 		db := c.PublicDB().(*database)
 
 		Convey("return Resemble RecordSchema on second call", func() {
-			err := db.Extend("note", skydb.RecordSchema{
+			_, err := db.Extend("note", skydb.RecordSchema{
```

Comment:
This test file tests `Extend`. At the very least you want to test the boolean return value from the `Extend` function.